### ISSUE TITLE
1.7.10 forge : minimum version required is 10.13.4.1566

### ISSUE THREAD

**Issue by slav37** [ForestryMC/ForestryMC#2043]
Thank you for reporting!

For issues:
 * Try the latest version, it may have fixed the issue already.
 * If you crashed, please paste the crash log to [gist](https://gist.github.com/) and link it here.
[
<img width="577" alt="screen shot 2018-03-15 at 12 16 49 am" src="https://user-images.githubusercontent.com/37392734/37444097-59b96d14-27e6-11e8-936b-dc8fbcf258db.png">
<img width="833" alt="screen shot 2018-03-15 at 12 16 07 am" src="https://user-images.githubusercontent.com/37392734/37444099-5bf637ba-27e6-11e8-8ea5-ef06219ddfda.png"> 

](url)

**Comment by MauveCloud** [comment]
Having a Forge installer jar for a _completely different version of Minecraft_ in your mods folder does not prove you have the right version of Forge _running_ in 1.7.10.  Please check your launcher profile to see what version of Forge is set up for 1.7.10.

**Comment by slav37** [comment]
I'm using a modpack on technic launcher. I'll try using forge 10.13.4.1566 but it says thats the minimum version, meaning i should be able to use the latest forge, no?

**Comment by slav37** [comment]
I get the same message. I even removed the newer forge from my mods folder
<img width="616" alt="screen shot 2018-03-15 at 11 35 42 am" src="https://user-images.githubusercontent.com/37392734/37473706-1356220e-2845-11e8-881b-c0ed621f9ede.png">

**Comment by MauveCloud** [comment]
Sorry, perhap I wasn't clear enough.  Please post an image of your profile settings in the launcher, or a link to a pastebin of fml-client-latest.log.  Putting files in the mods folder is how you install mods _under_ Forge, but that doesn't work for installing Forge itself.  Thus having the Forge installer jar there has _no bearing_ on what version of Forge is actually running.

**Comment by slav37** [comment]
[
<img width="647" alt="screen shot 2018-03-15 at 3 12 54 pm" src="https://user-images.githubusercontent.com/37392734/37485959-6ae5c894-2863-11e8-88f7-2ccca50e8533.png">
](url)

I think this is what you need?

**Comment by MauveCloud** [comment]
No.  I was asking for the _entire contents_ of the file, linked on a pastebin site (such as http://paste.ubuntu.com/), not just an image of the last few lines (although an image of the top few lines might technically be enough).

Alternatively, perhaps you could link me to the modpack you're using - is it one you're building yourself or one that someone else made?  I'm not familiar with the Technic launcher, but I tried installing it, and I don't see an easy way to tweak what version of Forge is being used - you might need to edit bin/version.json in a couple of places.

**Comment by slav37** [comment]
I'm using the Ark Survival Primitive+ modpack you can find on Technic Launcher. 
<img width="1007" alt="screen shot 2018-03-15 at 6 41 20 pm" src="https://user-images.githubusercontent.com/37392734/37494774-bc0e89aa-2880-11e8-9aad-96d8b6209ba6.png">

and whatever this is: 

[fml-client-latest.log](https://github.com/ForestryMC/ForestryMC/files/1817358/fml-client-latest.log)


 Thanks

**Comment by slav37** [comment]
The log is quite long just so you know.

**Comment by temp1011** [comment]
`[18:43:48] [Client thread/INFO] [FML/]: MinecraftForge v10.13.4.1558 Initialized`
You have an outdated forge version installed. Try updating the pack/ making a new instance I guess? This is not forestry though.

**Comment by MauveCloud** [comment]
I'm not happy with Technic - as far as I can tell, they don't provide a way to get a list of the mods in a modpack short of actually installing the modpack.  However, once I installed that modpack, I noticed it did not include any version of Forestry.  I don't know why you want to jeopardize the balance of an existing modpack by adding a new mod such as Forestry into the mix, but since the Technic launcher does not seem to provide a simple way to update Forge for a modpack, might I suggest switching to MultiMC?  Importing the modpack from the url didn't work, but once you install it via the Technic Launcher, you can copy the contents of the subfolders to a MultiMC instance and tweak things from there.

**Comment by slav37** [comment]
I have dropped plenty of new mods in the "mods" folder for the ark modpack but this is the only problem I came across. I have tried MultiMC but Idk what I am supposed to do once you open it.

**Comment by MauveCloud** [comment]
The modpack seems to be locked into using build 1558 - I tried editing bin/version.json to get it to use a newer build, and it reverted.  Admittedly, it's still the "recommended" build of Forge for 1.7.10, so on the one hand, it's somewhat questionable for mod authors to require a newer build (when developing for Minecraft 1.7.10), but on the other, the "latest" build (1614) for 1.7.10 is over 2 years old with no further changes, so I don't really understand why LexManos hasn't marked it as the new "recommended" build.  Looking at https://github.com/ForestryMC/ForestryMC/blob/4.1/src/main/java/forestry/Forestry.java#L54 it seems that Forestry 4.1 only requires build 1207 or newer, so unless you specifically need features from 4.2, you could use that version instead.

As far as MultiMC, it has always seemed intuitive to me, and I've been using it for a while, so I'm not sure how to explain things to someone who's used to a different launcher.

**Comment by slav37** [comment]
Thank you MauveCloud, I downloaded the 4.1 version and it worked. Finally!

**Comment by ambycloud** [comment]
if you are using the regular minecraft launcher, check to see what the Forge version says below the Play button. Mine was actually running 1558 when I had  version 1614 installed. I had to click the menu button in the upper left corner of the launcher. Then clicked Launcher options. Then I clicked on where it said Forge and you have the option of selecting which version of Forge you want to run. I selected the right version clicked save and restarted the launcher. Then no problems with forestry mod(yay!) Maybe this will help some folks.
